### ISSUE TITLE
Gestion erreur proposition d'un mot n°1

### ISSUE THREAD

**Issue by NicolasNotari** [maxime121294/Puissance4#4]
Lorsque nous faisons directement la proposition d'un mot et non d'une lettre seule, une erreur est comptée par lettre fausse ?

**Comment by MedhiBK** [comment]
Oui, une erreur sur une proposition compte autant qu'une erreur sur une lettre.

**Comment by NicolasNotari** [comment]
Ok donc si le mot est "tata" et que le joueur joue "momo", il y a 2 erreurs ?

**Comment by MedhiBK** [comment]
Non on lui compte une seule erreur, c'est comme si pour le mot "tata" il avait proposer la lettre C par exemple

**Comment by MedhiBK** [comment]
J'avais mal lu ton premier message : 
En gros que le joueur propose le mauvais mot ou une maivaise lettre, dans les deux cas on compte une seule erreur
